src/helix-ui/elements/HXTabsetElement.js

```diff
@@ -47,23 +46,23 @@ export class HXTabsetElement extends HXElement {
         });
     }
 
-    disconnectedCallback () {
+    $onDisconnect () {
         this.$tablist.removeEventListener('keyup', this._onKeyUp);
         this.$tablist.removeEventListener('keydown', this.$preventScroll);
         this.tabs.forEach(tab => {
             tab.removeEventListener('click', this._onTabClick);
         });
     }
 
-    attributeChangedCallback (attr, oldVal, newVal) {
+    $onAttributeChange (attr, oldVal, newVal) {
         if (!isNaN(newVal)) {
```

Comment:
logic should be wrapped in an attr guard (`if (attr === 'current-tab') { ... }`)